From dd95658f65d545ce3b355560b246521fb5d00498 Mon Sep 17 00:00:00 2001
From: Rohit Athavale <rathaval@xilinx.com>
Date: Fri, 23 Feb 2018 11:15:02 -0800
Subject: [PATCH 0301/1851] v4l: xilinx-vpss-scaler: Add YUV 420 to VPSS Scaler
 Driver

commit af4206dff63c5c56b383674f0d0530693951d357 from
https://github.com/Xilinx/linux-xlnx.git

This commit adds the YUV 420 media bus format to the
VPSS Scaler Driver.

Signed-off-by: Rohit Athavale <rathaval@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-vpss-scaler.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/platform/xilinx/xilinx-vpss-scaler.c b/drivers/media/platform/xilinx/xilinx-vpss-scaler.c
index 5fc41b940643..eb736290f847 100644
--- a/drivers/media/platform/xilinx/xilinx-vpss-scaler.c
+++ b/drivers/media/platform/xilinx/xilinx-vpss-scaler.c
@@ -1162,7 +1162,7 @@ xv_vscaler_setup_video_fmt(struct xscaler_device *xscaler, u32 code_in)
 	u32 video_in;
 
 	switch (code_in) {
-	case MEDIA_BUS_FMT_UYVY8_1_5X8:
+	case MEDIA_BUS_FMT_VYYUYY8_1X24:
 		dev_dbg(xscaler->xvip.dev,
 			"Vscaler Input Media Format YUV 420");
 		video_in = XVIDC_CSF_YCRCB_420;
@@ -1229,7 +1229,7 @@ static int xv_hscaler_setup_video_fmt(struct xscaler_device *xscaler,
 		vsc_out);
 
 	switch (code_out) {
-	case MEDIA_BUS_FMT_UYVY8_1_5X8:
+	case MEDIA_BUS_FMT_VYYUYY8_1X24:
 		dev_dbg(xscaler->xvip.dev,
 			"Hscaler Output Media Format YUV 420\n");
 		video_out = XVIDC_CSF_YCRCB_420;
-- 
2.31.1

